allensdk/brain_observatory/sync_dataset.py

```diff
@@ -108,11 +108,15 @@ def __init__(self, path):
         self._check_line_labels()
 
     def _check_line_labels(self):
-        deprecated_keys = set(self.line_labels) & self.DEPRECATED_KEYS
-        if deprecated_keys:
-            logger.info(f"The loaded sync file contains uses the following "
-                        f"deprecated line label keys: {deprecated_keys}. "
-                        f"Consider updating the sync file line labels.")
+        if hasattr(self, "line_labels"):
+            deprecated_keys = set(self.line_labels) & self.DEPRECATED_KEYS
+            if deprecated_keys:
+                logger.info(f"The loaded sync file contains uses the following "
+                            f"deprecated line label keys: {deprecated_keys}. "
+                            f"Consider updating the sync file line labels.")
+        else:
+            logger.info(f"The loaded sync file has no line labels and may not"
```

Comment:
Shouldn't this be a warning?

Reply:
@kschelonka I was worried a warning might come off as too 'severe' when loading sync files using `Dataset`. I've changed both messages to warnings.